appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -1141,6 +1141,10 @@ Blockly.ReplMgr.processRetvals = function(responses) {
         case "error":
             console.log("processRetVals: Error value = " + r.value);
             runtimeerr(escapeHTML(r.value) + Blockly.Msg.REPL_NO_ERROR_FIVE_SECONDS);
+        case "log":
```

Comment:
We need to add a `break;` at the end of the error case otherwise it will fall through here. Alternatively, move the `case "log":` and associated code up before the error branch, and add a `break;`